llvm/test/CodeGen/AArch64/cfi-annotation.ll

```diff
@@ -0,0 +1,112 @@
+; RUN: opt %s -o %t.bc
+; RUN: llvm-lto2 run %t.bc -r %t.bc,bar,px -r %t.bc,foo,px -r %t.bc,test,px \
+; RUN: -r %t.bc,fptr1,px -r %t.bc,fptr2,px -save-temps -o %t.o
+; RUN: llvm-dis %t.*.opt.bc
```

Comment:
Is it possible to test this directly using `opt -passes=lowertypetests`, instead of going through llvm-lto2? Similar to all the other tests in llvm/test/Transforms/LowerTypeTests.